bin/index.js

```diff
@@ -416,10 +519,25 @@ const promptTopLevelFields = (manifest) => {
 
         return true;
       }
+    },
+    {
+      type: 'list',
+      name: 'repositoryType',
+      message: 'What type of repository you want to use?',
```

Comment:
"What type of repository **do** you want to use?"

Reply:
Done